SCP fight club/Assets/Scripts/Movement.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Movement : MonoBehaviour
+{
+    [SerializeField]
+    private float accelForce = 10f;
+
+    private new Rigidbody2D rigidbody;
+    public float speed = 1.0f;
+
+    void Start()
+    {
+        rigidbody = GetComponent<Rigidbody2D>();
+    }
+
+    private void Update()
+    {
+        // wanted to use "if (Input.GetAxisRaw("Horizontal") > 0);" but I don't know how
+
+         if(Input.GetKey(KeyCode.RightArrow))
```

Comment:
I think there's a space on this line before the start of the if statement. Might wanna get rid of that for style rules.